build/gulpfile.sql.js

```diff
@@ -113,25 +114,23 @@ gulp.task('install-sqltoolsservice', () => {
 	return installService();
 });
 
-function installSsmsMin() {
+gulp.task('install-ssmsmin', () => {
 	const config = require('../extensions/admin-tool-ext-win/config.json');
-	return platformInfo.getCurrent().then(p => {
-		const runtime = p.runtimeId;
-		// fix path since it won't be correct
-		config.installDirectory = path.join(__dirname, '..', 'extensions', 'admin-tool-ext-win', config.installDirectory);
-		var installer = new serviceDownloader(config);
-		const serviceInstallFolder = installer.getInstallDirectory(runtime);
-		const serviceCleanupFolder = path.join(serviceInstallFolder, '..');
-		console.log('Cleaning up the install folder: ' + serviceCleanupFolder);
-		return del(serviceCleanupFolder + '/*').then(() => {
-			console.log('Installing the service. Install folder: ' + serviceInstallFolder);
-			return installer.installService(runtime);
-		}, delError => {
-			console.log('failed to delete the install folder error: ' + delError);
-		});
+	const runtime = 'Windows_64';
```

Comment:
Please add a comment here explaining that this extension is purposely windows-only